elife/logrotate.sls

```diff
@@ -0,0 +1,5 @@
+custom daily logrotate script:
+    file.managed:
+        - name: /etc/cron.daily/logrotate
+        - source: salt://elife/config/etc-cron.daily-logrotate
+        - mode: 755
```

Comment:
does this need a `require` to be done after `logrotate` installation, which would presumably also create this cron on new instances?

Reply:
not sure, wouldn't have thought so but I've added it anyway